src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -188,7 +189,7 @@ private Optional<ParserResult> importFile(Path file, String importFormat) {
         }
     }
 
-    public void processArguments() {
+    public void processArguments() throws FileNotFoundException {
```

Comment:
Please add JavaDoc `@throws` to state when this exception is thrown.

It is strange that upon startup of JabRef something is happening.

The issue itself has two headings "First implementation" and "Follow-up implementation". I cannot relate something happening on startup at each of the headings. Maybe you can explain?